azure-pipelines.yml

```diff
@@ -442,7 +442,7 @@ jobs:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
   - bash: |
-      set -e
```

Comment:
I think "-e" is effectively checking that each of these steps works and without it this action will pass as long as the last command does, so you could imagine a situation in which only the `//hw/bitstream/vivado:hyperdebug` target gets generated and copied, and this job thinks it passed.

Reply:
You might want to trap the exit event (like we do in run_fpga_test.sh) with a command that copies over the logs to ensure we get them even when tests fail.

Reply:
Ah you're right -- this was in the wrong place. I should have removed it from the step below.